include/nana/c++defines.hpp

```diff
@@ -59,6 +59,7 @@
 
 #		define constexpr const	//no support of constexpr until Visual C++ 2015 ? const ??
 #	endif
+#	undef STD_FILESYSTEM_NOT_SUPPORTED
```

Comment:
Only undef STD_FILESYSTEM_NOT_SUPPORTED for VS2015, because VS2013 provides a <filesystem> which is not defined in namespace std::experimental.